spec/NewTwitchApi/Resources/UsersApiSpec.php

```diff
@@ -10,98 +10,110 @@
-        $this->getUsersFollows("TEST_TOKEN", null, null, null, 42)->shouldBeAnInstanceOf(ResponseInterface::class);
+        $this->getUsersFollows('TEST_TOKEN', null, null, null, 42)->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
-    function it_should_get_users_follows_by_everything(Client $guzzleClient, Response $response)
+    public function it_should_get_users_follows_by_everything(Client $guzzleClient, Response $response)
     {
         $guzzleClient->send(new Request('GET', 'users/follows?from_id=12345&to_id=98765&first=42&after=99', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
-        $this->getUsersFollows("TEST_TOKEN", '12345', '98765', 42, 99)->shouldBeAnInstanceOf(ResponseInterface::class);
+        $this->getUsersFollows('TEST_TOKEN', '12345', '98765', 42, 99)->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
-    function it_should_get_user_with_access_token(Client $guzzleClient, Response $response)
+    public function it_should_get_user_with_access_token(Client $guzzleClient, Response $response)
     {
-        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer access-token']))->willReturn($response);
-        $this->getUsers("access-token")->shouldBeAnInstanceOf(ResponseInterface::class);
+        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->getUsers('TEST_TOKEN')->shouldBeAnInstanceOf(ResponseInterface::class);
     }
 
-    function it_should_get_user_with_access_token_convenience_method(Client $guzzleClient, Response $response)
+    public function it_should_get_user_with_access_token_convenience_method(Client $guzzleClient, Response $response)
     {
-        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer access-token']))->willReturn($response);
-        $this->getUserByAccessToken('access-token')->shouldBeAnInstanceOf(ResponseInterface::class);
+        $guzzleClient->send(new Request('GET', 'users', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->getUserByAccessToken('TEST_TOKEN')->shouldBeAnInstanceOf(ResponseInterface::class);
+    }
+
+    public function it_should_create_a_follow(Client $guzzleClient, Response $response)
+    {
+        $guzzleClient->send(new Request('POST', 'users/follows?from_id=123&to_id=321', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
+        $this->createUserFollow('TEST_TOKEN', '123', '321')->shouldBeAnInstanceOf(ResponseInterface::class);
+    }
+
+    public function it_should_create_a_follow_with_notifications(Client $guzzleClient, Response $response)
+    {
+        $guzzleClient->send(new Request('POST', 'users/follows?from_id=123&to_id=321&allow_notifications=true', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
```

Comment:
This spec is failing because the generated URL has `&allow_notificactions=1` but the spec here is expecting `allow_notifications=true`.